cdap-watchdog/src/test/java/io/cdap/cdap/logging/appender/ErrorCodeProviderLoggingTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright © 2014-2019 Cask Data, Inc.
```

Comment:
2022

Reply:
There is no Cask data now right? For most files it is 2019. Does it mean we are out of the copyright? So I didn't worry about this much.

Reply:
There is. Please keep the year update to date.

Reply:
Done.